src/roswire/ros2/node.py

```diff
@@ -1,29 +1,62 @@
 # -*- coding: utf-8 -*-
 __all__ = ('ROS2Node',)
 
-import typing
+from loguru import logger
```

Comment:
Sorry to be super pedantic: `loguru` and `dockerblade` should go together in a second block of imports. (See comment below.)